js/src/components/account-card/index.js

```diff
@@ -22,22 +22,33 @@ import './index.scss';
  */
 export const APPEARANCE = {
 	GOOGLE: 'google',
+	GOOGLE_ADS: 'google-ads',
 	PHONE: 'phone',
 	ADDRESS: 'address',
 };
 
+const googleLogo = (
+	<img
+		src={ googleLogoURL }
+		alt={ __( 'Google Logo', 'google-listings-and-ads' ) }
+		width="40"
+		height="40"
+	/>
+);
+
 const appearanceDict = {
 	[ APPEARANCE.GOOGLE ]: {
-		icon: (
-			<img
-				src={ googleLogoURL }
-				alt={ __( 'Google Logo', 'google-listings-and-ads' ) }
-				width="40"
-				height="40"
-			/>
-		),
+		icon: googleLogo,
 		title: __( 'Google account', 'google-listings-and-ads' ),
 	},
+	[ APPEARANCE.GOOGLE_ADS ]: {
+		icon: googleLogo,
+		title: __( 'Google Ads', 'google-listings-and-ads' ),
+		defaultDescription: __(
```

Comment:
I added it to avoid repeating the same `description` string across many AccountCard usages, and I actually got stuck here for a while when naming it between `description` vs others. 😂 

> ❔ Also, on the consumer perspective, it starts to feel a bit funny because the consumer can provide description via `description` props, or `appearance` props like `appearance={{ description: '' }}`.

I ended up with `defaultDescription` for the same reason. Because it's a bit vague to set card description with `props.description` and `appearance.description`.

---

There's a new idea coming to me but it will change larger scopes than this PR. I will submit another PR for us to discuss.

Reply:
> There's a new idea coming to me but it will change larger scopes than this PR. I will submit another PR for us to discuss.

Ref: #1120